hack/run-gendocs.sh

```diff
@@ -45,8 +45,7 @@ kube::util::gen-doc "${genman}" "${KUBE_ROOT}" "docs/man/man1"
 kube::util::gen-doc "${genbashcomp}" "${KUBE_ROOT}" "contrib/completions/bash/"
 kube::util::gen-analytics "${KUBE_ROOT}"
 
-"${mungedocs}" "--root-dir=${KUBE_ROOT}/docs/"
-ret=$?
+"${mungedocs}" "--root-dir=${KUBE_ROOT}/docs/" && ret=0 || ret=$?
```

Comment:
Yeah, I knew it wasn't working right previously, just didn't bother to fix. Thanks for fixing. :)